src/sql/workbench/browser/actions.ts

```diff
@@ -88,3 +89,20 @@ export class ConfigureDashboardAction extends Task {
 		accessor.get(IOpenerService).open(URI.parse(ConfigureDashboardAction.configHelpUri));
 	}
 }
+
+export class ClearSavedAccountsAction extends Task {
```

Comment:
We can have saved accounts that don't show up in the account picker anymore because their provider doesn't exist anymore. This gives users a way to just reset their accounts and start fresh. It's also useful as a debugging step to tell the user to just run this command.